opta/commands/deploy.py

```diff
@@ -118,6 +119,7 @@ def deploy(
         amplitude_client.DEPLOY_EVENT,
         event_properties={"org_name": layer.org_name, "layer_name": layer.name},
     )
+    _, is_auto = __check_layer_and_image(layer, image)
```

Comment:
I didn't realize we would not need `image_name` then we can just simply have this function returning only `is_auto`

Reply:
Changed.